### PR TITLE
Enable serverextension after env is copied

### PR DESCRIPTION
**Pull request by pindge**
Iframe error is gone from the console
` jupyterlab_code_formatter` don't need `--sys-prefix`

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -36,9 +36,6 @@ RUN echo "Adding jupyter lab extensions" \
   && jupyter labextension install --no-build ipycanvas@v0.3.4 \
   && jupyter labextension install --no-build jupyterlab_iframe@0.2.1 \
   && jupyter lab build \
-  && jupyter serverextension enable --py jupyterlab_code_formatter --sys-prefix \
-  && jupyter serverextension enable --py --system jupyterlab_iframe \
-  && jupyter labextension list \
```
Comment (Kirill888) [inline docker/Dockerfile]: please keep labextension list, it's useful for reviewing build logs

File: docker/Dockerfile [hunk 2]
```diff
@@ -103,7 +100,7 @@ RUN useradd -m -s /bin/bash -N -g $nb_gid -u $nb_uid $nb_user
 # Copy python env
 ARG py_env_path
 COPY --chown=1000:100 --from=env_builder $py_env_path $py_env_path
-
+
```
Comment (Kirill888) [inline docker/Dockerfile]: space?

File: docker/Dockerfile [hunk 3]
```diff
@@ -115,6 +112,8 @@ RUN mkdir -p /var/share/TPX08_atlas_compact/ \
 # Patch env when needed here
 RUN pip install --no-cache-dir nbresuse \
   && jupyter serverextension enable --py --system nbresuse \
+  && jupyter serverextension enable --py jupyterlab_code_formatter \
```
Comment (Kirill888) [inline docker/Dockerfile]: Please make them all `--system`.
These just write some yaml files, `--system` writes for "all users" somewhere in `/` and without writes in home folder, but home folder is overwritten by volume during deployment, so has to be system

Comment (pindge) [reply docker/Dockerfile]: ok

File: docker/Dockerfile (hunk 1 above)
Comment (tom-butler) [inline docker/Dockerfile]: Can we please keep this line? it's useful for identifying what extensions are installed in build logs.

Comment (pindge) [reply docker/Dockerfile]: Fixed

Comment (pindge) [reply docker/Dockerfile]: Fixed

**Pull request merged**